src/relays/TrustedRelay.sol

```diff
@@ -55,6 +59,10 @@ contract TrustedRelay {
     TokenLike              public immutable dai;
     WormholeOracleAuthLike public immutable oracleAuth;
     WormholeJoinLike       public immutable wormholeJoin;
+    DsValueLike            public immutable ethPriceOracle;
+    uint256                public immutable gasMargin; // in RAY (e.g 150% = 1.5 * RAY)
```

Comment:
Should this be immutable? Maybe we will want to adjust?

Reply:
I think this should be immutable otherwise the gasFee check could easily be avoided by the trusted signer who could bundle a relay() call with a setGasMargin() call. I guess the way to adjust the margin would be to redeploy another instance of TrustedRelay.

Reply:
The trusted signer should not be a `ward` though. That should just be set to Maker governance.

Reply:
Ah yes you're right.

Reply:
Note that I was thinking to deploy one TrustedRelay per integration and let each integrator add/remove signers so as to insulate the impact of a compromised signer and to offer greater flexibility to integrators. In that case, we'd need two wards: one for the integrator and one for Maker governance.